<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PermissionsDialog</class>
 <widget class="QDialog" name="PermissionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Permissions for new folders and files</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
font-size: 11px;
}

#lFolders, #lFiles
{
  font-family: &quot;HelveticaNeue-Light&quot;;
  font-size: 12px;
}

QGroupBox
{
margin-top: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lPermissionsDesc">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Files and folders have an owner and belongs to a group of users. Using this dialog, you can grant different access permission for new files and folders created by MEGAsync to users that belong to the same group and to the rest of the users in the computer. If you don't need to change them, please leave them as default for security reasons.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="wFolderPermissions">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Folders</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lGroupFolders">
        <property name="text">
         <string>Group permissions:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PermissionsWidget" name="wFolderGroup" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lPublicFolders">
        <property name="text">
         <string>Public permissions:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PermissionsWidget" name="wFolderPublic" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lFolderPermissionsDesc">
        <property name="text">
         <string>Numeric value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lFolderPermissions"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="wFilePermissions">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lOwnerFiles">
        <property name="styleSheet">
         <string notr="true">font: normal;</string>
        </property>
        <property name="text">
         <string>Owner permissions:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PermissionsWidget" name="wFileOwner" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lGroupFiles">
        <property name="text">
         <string>Group permissions:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PermissionsWidget" name="wFileGroup" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lPublicFiles">
        <property name="text">
         <string>Public permissions:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PermissionsWidget" name="wFilePublic" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lFilePermissionsDesc">
        <property name="text">
         <string>Numeric value:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lFilePermissions"/>
      </item>
     </layout>
     <zorder>lFilePermissionsDesc</zorder>
     <zorder>lFilePermissions</zorder>
     <zorder>lGroupFiles</zorder>
     <zorder>lPublicFiles</zorder>
     <zorder>wFileGroup</zorder>
     <zorder>wFilePublic</zorder>
     <zorder>lOwnerFiles</zorder>
     <zorder>wFileOwner</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PermissionsWidget</class>
   <extends>QWidget</extends>
   <header>PermissionsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PermissionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PermissionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
